lib/text/ttml_text_parser.js

```diff
@@ -933,6 +950,28 @@ shaka.text.TtmlTextParser = class {
 
     return (milliseconds / 1000) + seconds + (minutes * 60) + (hours * 3600);
   }
+
+  /**
+   * If ttp:cellResolution provided returns cell resolution info
+   * with number of columns and rows into which the Root Container
+   * Region area is divided
+   *
+   * @param {?string} cellResolution
+   * @return {?Object}
+   * @private
+   */
+  static getCellResolution_(cellResolution) {
+    const matches = /^(\d+) (\d+)$/.exec(cellResolution);
```

Comment:
cellResolution can be null, and this would convert `null` into the string `"null"`.

While that shouldn't cause any false positives, it may be best to handle `!cellResolution` explicitly, either here or when the attribute is extracted from the TTML.